docs/reference/ocm_add_componentversions.md

```diff
@@ -160,28 +160,6 @@ are configured for the operation. It has the following format
 </center>
 
 The uploader name may be a path expression with the following possibilities:
-  - <code>ocm/npmPackage</code>: uploading npm artifacts
-
-    The <code>ocm/npmPackage</code> uploader is able to upload npm artifacts
-    as artifact archive according to the npm package spec.
```

Comment:
Hmm, why has the order changed?  may be there is a sort missing for the generator to achieve a stable order.

yes, in cmds/ocm/commands/common/options/uploaderoption/option.go (Usage) the list should be sorted by the Name attribute.
Similar problem in downloaderoption.

Reply:
I'll add the sort 👍

Reply:
I'm already doing some fixes, should I add it?

Reply:
Yes please, thank you.